src/Team-Services-Bot.AcceptanceTests/Team-Services-Bot.AcceptanceTests.csproj

```diff
@@ -34,6 +34,7 @@
     <DocumentationFile>bin\Debug\Vsar.TSBot.AcceptanceTests.xml</DocumentationFile>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <CodeAnalysisIgnoreGeneratedCode>true</CodeAnalysisIgnoreGeneratedCode>
+    <NoWarn>1591</NoWarn>
```

Comment:
I prefer disabling such things in the code (#pragma warning disable) as the test class library might have helper classes that require XML doc.